app/views/tasks/_task.html.haml

```diff
@@ -1,28 +1,23 @@
-.grid-row.task-row{id: "task-#{task.id}"}
-  .column-one-third.task-details
-    %h2.heading-medium
-      = task.description
-    %p
-      = task_status(task)
-
-  - case task.status
-  - when 'completed'
-    .column-two-thirds.task-controls
-  - when 'in_progress'
-    .column-two-thirds.task-controls
-      .grid-row
-        .column-one-third
-          = link_to 'Download template', framework_template_path_for(task), class: 'button button-secondary'
-        .column-two-third
-          = link_to 'View submission status', task_submission_path(task_id: task.id, id: task.latest_submission.id), class: 'button'
-  - else
-    .column-two-thirds.task-controls
-      .grid-row
-        .column-one-third
-          = link_to 'Download template', framework_template_path_for(task), class: 'button button-secondary'
-        .column-one-third
-          = link_to 'Upload submission', new_task_submission_path(task_id: task.id), class: 'button'
-        .column-one-third
-          = link_to 'Report no business', new_task_no_business_path(task_id: task.id), class: 'button'
-          %span.button-hint
-            Previously called nil return
+%section.ccs-task{id: "task-#{task.id}"}
+  %header.ccs-task__information
+    .ccs-task__body
+      %h3.govuk-heading-m
+        = task.framework.short_name
+        = task.framework.name
+      %h4.govuk-heading-s
+        Report management information for
+        = task_due_month(task)
```

Comment:
Shouldn't this be based on `period_month` / `period_year` rather than the due date? (i.e. user reports management information for August, *in September* for example)

Reply:
Yes, there is `task_due_month(@task)` we need `task_for_month(@task)` or similar.

Reply:
very good point! I can update this.

Reply:
this has been updated to show the correct month for the task.

Reply:
:+1: